### ISSUE TITLE
Pokemon Heartgold White Screens after soft reset

### ISSUE THREAD

**Issue by kellyGipson** [DS-Homebrew/TWiLightMenu#1967]
**Flashcard or Exploits?** 
R4i SHDC 2022(Ace3ds+ clone from r4ishdc.com.cn)
**Base Console:** 
new 2ds XL, ds lite(happens on any ds/2/3ds I presume).

**TWiLight Menu++ Version:** 
v24.12.1
**TWiLight Menu++ Theme:** 

### Steps to reproduce

1.
2.
3.

**Comment by RocketRobz** [comment]
Is nds-bootstrap turned on in TWLMenu++ Settings?

**Comment by kellyGipson** [comment]
Yes it is. I just updated my ticket. I fat fingered the submit button, my apologies.
